### DISCUSSION TITLE
Allow Velocity /Server command to execute anyway

### DISCUSSION THREAD

**Discussion by Strahilchu** [Zrips/CMI#6347] [Feature Requests]
**CMIB does not work on Velocity.

Although /Server Lobby functions, CMI will not let me run the command.
I have tried to get CMI to sudo a player as console and still will not allow it.

It's okay if CMI does not support Velocity yet, however there is something stopping it from working even further.

My request is to allow CMI to execute /Server Lobby regardless if it's on a Bungee network, since the command works, there is no reason to not allow it.**

**Comment by CloudeLecaw** [comment]
https://github.com/Zrips/CMI/discussions/2515

**Comment by mrfloris** [comment]
https://www.spigotmc.org/resources/cmi-298-commands-insane-kits-portals-essentials-economy-mysql-sqlite-much-more.3742/update?update=521168

There is a mention in the changelog for velocity support in today's CMI release.
You can get the free cmiv jar file here: https://www.zrips.net/cmiv/
